ArtOfIllusion/src/artofillusion/CameraFilterDialog.java

```diff
@@ -59,14 +59,7 @@ public CameraFilterDialog(WindowWidget parent, Scene scene, SceneCamera camera,
     FormContainer content = new FormContainer(2, 2);
     setContent(content);
     LayoutInfo fillLayout = new LayoutInfo(LayoutInfo.CENTER, LayoutInfo.BOTH, null, null);
-    filtersPanel = new FiltersPanel(theCamera, new Runnable()
-    {
-      @Override
-      public void run()
-      {
-        applyFilters();
-      }
-    });
+    filtersPanel = new FiltersPanel(theCamera, this::applyFilters);
     BorderContainer previewPanel = new BorderContainer();
```

Comment:
This change isn't related to compiler warning cleanups. While I do prefer making this cleanup, it belongs in a separate commit.